docs/modules/pipelines/partials/nav.adoc

```diff
@@ -29,6 +29,7 @@
 **** xref:pipelines:kinesis.adoc[]
 **** xref:pipelines:pulsar.adoc[]
 ** xref:pipelines:serialization.adoc[]
+** xref:pipelines:compute-isolation.adoc[]
```

Comment:
We will of course need to update this adoc file name to `job-placement-control` presumably 😄

Reply:
Yup, I had done that - now to check that hasn't broken anything else ;-)